### PR TITLE
examples : use event.code instead of event.keyCode

### PR DESCRIPTION
**Pull request by felixmariotto**
following #21055

> Issue : the pointerlock example, FirstPersonControls and FlyControls checked the keyCode property of the keydown and keyup events, which means it was bound to characters. Some keyboard layouts, like the French one displayed bellow, don't map the WASD keys at the same place, whereas we actually play with the same keys as everybody, only these keys represent different characters.
>
> Screenshot_1

This PR continues to fix this issue in various examples by using [event.code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code), which is layout-independant.

### CODE REVIEW

**Comment by Mugen87** [comment]
https://raw.githack.com/felixmariotto/three.js/dev-key-events-fix2/examples/index.html

**Comment by Mugen87** [comment]
Since the has not official stopped support for IE11, it's worth mentioning that `KeyboardEvent.code` is not supported by IE11.

https://caniuse.com/keyboardevent-code

**Review by Mugen87 — APPROVED** [review]

File: examples/js/controls/FirstPersonControls.js [hunk 1]
```diff
@@ -149,45 +149,45 @@ THREE.FirstPersonControls = function ( object, domElement ) {
 
 		//event.preventDefault();
 
-		switch ( event.keyCode ) {
+		switch ( event.code ) {
 
-			case 38: /*up*/
-			case 87: /*W*/ this.moveForward = true; break;
+			case 'ArrowUp': /*up*/
+			case 'KeyW': /*W*/ this.moveForward = true; break;
```
Comment (donmccurdy) [inline examples/js/controls/FirstPersonControls.js:155]: minor: could remove the inline comments since `event.code` values are more readable.

**Review by donmccurdy — APPROVED** [review]

File: examples/js/controls/FirstPersonControls.js (hunk 1 above)
Comment (felixmariotto) [reply examples/js/controls/FirstPersonControls.js:155]: ok I'm doing this.
So should I push a new commit or modify the existing one and force-push ?
Pushing updates after changes were approved seemed to be an issue... Sorry for the confusion.

Comment (Mugen87) [reply examples/js/controls/FirstPersonControls.js:155]: > Pushing new commits after changes were approved seemed to be a problem

I would say changing code after a review is problematic no matter if you add new commits or force-push. 

In this particular case (where you just remove obsolete comments), you can do as you like.

Comment (Mugen87) [reply examples/js/controls/FirstPersonControls.js:155]: PS: You might also want to remove these comments in #21055.

Comment (felixmariotto) [reply examples/js/controls/FirstPersonControls.js]: @Mugen87 ok, I've made the changes with a new commit in each PR

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**